observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="observability-ui-plugins-overview"]
+= Observability UI plugins overview
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
```

Comment:
Suggest adding an attribute for the Operator name if there isn't one already

Reply:
coo-full is there already - will use it here

Reply:
done